content/docs/2.13/authentication-providers/aws.md

```diff
@@ -0,0 +1,140 @@
++++
+title = "Pod Identity Webhook for AWS by using IAM Roles for Service Accounts (IRSA)"
```

Comment:
```suggestion
title = "EKS Pod Identity Webhook for AWS by using IAM Roles for Service Accounts (IRSA)"
```

Just a nit to group all AWS/EKS related auth providers together

Reply:
This works outside EKS too, so I prefer to not include EKS prefix. Let me rename AWS auths from "XXXXX for AWS" to "AWS XXXX"

Reply:
Now they are grouped: 
![image](https://github.com/kedacore/keda-docs/assets/36899226/92237eda-ab34-405a-b141-ab6159de1ae4)

Reply:
WDYT @zroubalik ?

Reply:
Agreed